src/libs/page-detect.js

```diff
@@ -46,6 +46,8 @@ export const isCommit = () => isSingleCommit() || isPRCommit() || (isPRFiles() &
 
 export const isCompare = () => isRepo() && /^\/compare/.test(getRepoPath());
 
+export const isQuickPR = () => isCompare() && /\?quick_pull=1$/.test(location.href);
```

Comment:
Perhaps the second part should be:

```js
/[?&]quick_pull=1(&|$)/.test(location.search)
```

Reply:
Right! Did so in additional commit, thanks!